scripts/utils.js

```diff
@@ -19,10 +19,10 @@ export const [setLibs, getLibs] = (() => {
     (prodLibs, location) => {
       libs = (() => {
         const { hostname, search } = location || window.location;
-        if (!(hostname.includes('.hlx.') || hostname.includes('local'))) return prodLibs;
+        if (!(hostname.includes('.aem.') || hostname.includes('local'))) return prodLibs;
```

Comment:
These are here just to be able to start the federal project, right? As in: a project still using `hlx` and fetching federal content will not be impacted by the fact that we're just using `aem` here.

Reply:
Yup, this is just to test links within the scope of the federal project itself. Other than that, content is always fetched as `.plain.html` which can be done both from hlx or aem